### DISCUSSION TITLE
url: /fishaudio/speech-lm-v1/resolve/main/config.json 这个文件在哪呢，没有找到

### DISCUSSION THREAD

**Discussion by couldn** [fishaudio/fish-speech#112] [Q&A] [answered]
requests.exceptions.ConnectionError: (MaxRetryError("HTTPSConnectionPool(host='huggingface.co', port=443): Max retries exceeded with url: /fishaudio/speech-lm-v1/resolve/main/config.json

**Comment by wonder-perfect** [comment, marked as answer]
这是网络连接问题，挂梯子，hf不能直连
